presto-docs/src/main/sphinx/functions/list.rst

```diff
@@ -95,6 +95,7 @@ C
 - :func:`color`
 - :func:`combinations`
 - :func:`concat`
+- :func:`concat_ws`
```

Comment:
is it possible to test this automatically?

Reply:
I think @mosabua and @electrum tried that, but I have no idea... I just found this while looking up the docs :)

Reply:
It might be  .. we never had time to look into this in more detail. Currently we maintain it manually. The list is also not complete (nor is the docs). There are issues with the func syntax usage, multiple argument combinations and this page having operators and other stuff in it as well.